Failed to build with Bash due invalid echo command usage in version.mk

I have tried to build Teleport from Arch Linux/Bash and coreutils 8.25:

```
$ make
make -f version.mk setver
make[1]: Entering directory '/home/sergey-zolotorev/go/src/github.com/gravitational/teleport'
<standard input>:1:61: illegal character U+005C '\'
make[1]: *** [version.mk:19: setver] Error 2
make[1]: Leaving directory '/home/sergey-zolotorev/go/src/github.com/gravitational/teleport'
make: *** [Makefile:115: setver] Error 2

```

It caused by invalid usage of echo command. Echo from coreutils 8.25 does not expand escape sequences automatically (e.g. `echo "foo\nbar"` will print 'foo\nbar').

Solution is to add `-e` option for echo: `@echo -e $(VERSION_GO) | gofmt > version.go` and the same for gitref.go but it probably can broke build in other shells where echo is builtin command or with other versions of echo which does not support `-e` option.

--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 # Naming convention:
 #	for stable releases we use "1.0.0" format
 #   for pre-releases, we use   "1.0.0-beta.2" format
-VERSION=1.0.0
+VERSION=1.0.1-beta.1
 
 # These are standard autotools variables, don't change them please
 BUILDDIR ?= build

--- a/version.go
+++ b/version.go
@@ -2,7 +2,7 @@
 package teleport
 
 const (
-	Version = "1.0.0"
+	Version = "1.0.1-beta.1"
 )
 
 var Gitref string

--- a/version.mk
+++ b/version.mk
@@ -16,5 +16,5 @@ func init() { Gitref = \"$(GITREF)\"}  "
 #
 .PHONY:setver
 setver:
-	@echo $(VERSION_GO) | gofmt > version.go
-	@echo $(GITREF_GO) | gofmt > gitref.go
+	@printf $(VERSION_GO) | gofmt > version.go
+	@printf $(GITREF_GO) | gofmt > gitref.go

